src/mina-signer/src/test-vectors/legacySignatures.ts

```diff
@@ -91,6 +91,62 @@ let strings = [
  * - the 3 strings.
  */
 let signatures: { [k: string]: { field: string; scalar: string }[] } = {
+  devnet: [
```

Comment:
nit: we could save a few lines here since devnet and testnet are the same payload anyway